include/zephyr/fs/zms.h

```diff
@@ -18,45 +18,43 @@ extern "C" {
 #endif
 
 /**
- * @brief Zephyr Memory Storage (ZMS)
  * @defgroup zms Zephyr Memory Storage (ZMS)
  * @ingroup file_system_storage
  * @{
  * @}
  */
 
 /**
- * @brief Zephyr Memory Storage Data Structures
  * @defgroup zms_data_structures Zephyr Memory Storage Data Structures
```

Comment:
Do not capitalize `Data` and `Structures` to make it more clear what makes up an acronym and what is just normal words?